ui/tests/acceptance/secrets/backend/aws/aws-configuration-test.js

```diff
@@ -77,15 +78,12 @@ module('Acceptance | aws | configuration', function (hooks) {
   });
 
   test('it should save root AWS configuration', async function (assert) {
-    assert.expect(3);
```

Comment:
removing assert.expect unless using mirage. To my knowledge these are only helpful if an assert could go unchecked. this use case only seems to happen when using something like `this.server.get` and the get is never hit.